tools/tasks/scriptTasks.js

```diff
@@ -30,6 +32,14 @@ module.exports = function(gulp, config) {
         .pipe(gulp.dest(OPTIONS.DIR.BUILD_SCRIPTS));
```

Comment:
you are correct, this (and the failing tests) are updated in #227 .

I'll pull that in for fun, but this PR will be closed soon cause its just an experiment. :-)